avalon/nuke/pipeline.py

```diff
@@ -165,54 +154,37 @@ class Creator(api.Creator):
     """
     node_color = "0xdfea5dff"
 
-    def __init__(self, *args, **kwargs):
```

Comment:
Not sure if this was important?

Reply:
I wasn't able to make it work with this on. We have further changes to this in our code and it was quite a stopper.